Jenkinsfile

```diff
@@ -1,64 +1,93 @@
-pipeline {
-    agent none
-    environment {
-        ISISROOT="${workspace}" + "/build/"
-        ISIS3TESTDATA="/usgs/cpkgs/isis3/testData/"
-        ISIS3DATA="/usgs/cpkgs/isis3/data/"
+properties([pipelineTriggers([githubPush()])])
+
+def nodes = [:]
+
+nodes["isis-fedora-25"] = {
+    node("isis-fedora-25") {
+        withEnv(["ISISROOT=" + "${workspace}" + "/build/", "ISIS3TESTDATA=/usgs/cpkgs/isis3/testData/", "ISIS3DATA=/usgs/cpkgs/isis3/data/"]) {
+            stage ("Fedora 25") {
+                git branch: 'dev', url: 'https://github.com/USGS-Astrogeology/ISIS3.git'
+                sh """
+                    conda env create -n isis3 -f environment.yml
```

Comment:
We will need an additional line to pull from the gtest submodule

Reply:
Done